src/Illuminate/Redis/Connections/PhpRedisConnection.php

```diff
@@ -4,6 +4,9 @@
 
 use Closure;
 
+/**
+ * @mixin \Predis\Client
```

Comment:
/**
     * Create a new Predis connection.
     *

It is a client, the Connection class says 

    /**
     * The Predis client.
     *
     * @var \Predis\Client
     */
    protected $client;

But the PhpRedisConnection says ... both?

    /**
     * Create a new Predis connection.
     *
     * @param  \Redis  $client
     * @return void
     */
    public function __construct($client)
    {
        $this->client = $client;
    }